core/spec/models/spree/payment_method_spec.rb

```diff
@@ -395,4 +395,22 @@ def gateway_class
       expect(described_class::DISPLAY).to eq([:both, :front_end, :back_end])
     end
   end
+
+  describe "#find_sti_class" do
+    context "with an unexisting type" do
+      context "while retrieving records" do
+        let!(:unsupported_payment_method) { create(:payment_method, type: 'UnsupportedPaymentMethod') }
+        before do
+          unsupported_payment_method.update type: 'UnsupportedPaymentMethod'
```

Comment:
I think this `before` block has no effect on the record, as it's updating the type but the value is the same as the initial one from line 402.

Reply:
Right, I removed the block.